src/main/java/entities/Review.java

```diff
@@ -0,0 +1,34 @@
+package entities;
+
+public class Review {
+    private float rating;
+    private String comment;
+    private static int count = 0;
+    private int id;
+
+    /**
+     * Create a new Review with rating, comment, and its unique id (start from 1)
+     *
+     * @param rating the rating of this single Review.
+     * @param comment the comment of this single Review.
+     */
+
+    public Review(float rating, String comment) {
+        this.rating = rating;
+        this.comment = comment;
+        this.id = count++;
+    }
+
+    public int getId() {
+        return id;
+    }
+
```

Comment:
It would be good if we can have a getter method for getting the rating of this review. Because we may want to calculate the average rating of a user.